From 37901b49c5de89ff06e1e84df6f98a4e0272f431 Mon Sep 17 00:00:00 2001
From: Kyle McMartin <kmcmarti@redhat.com>
Date: Tue, 6 May 2014 05:12:54 -0400
Subject: [redhat] kernel.spec: use correct %dist in kernel-devel post

Message-id: <20140506051253.GI674@redacted.bos.redhat.com>
Patchwork-id: 79403
O-Subject: [RHEL7 PATCH] kernel.spec: use correct %dist in kernel-devel post
Bugzilla:
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Josh Boyer <jwboyer@redhat.com>

Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=7419194

Don Dutile noticed this earlier today... we're not correctly hardlinking
together multiple kernel-devel tree installs. Granted, we could probably
just remove this snippet, but it's mostly harmless and reduces the disk
footprint of Z-stream kernel-devel.

Signed-off-by: Kyle McMartin <kmcmarti@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 7eff2d3..761b3d1 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1249,7 +1249,7 @@ if [ "$HARDLINK" != "no" -a -x /usr/sbin/hardlink ]\
 then\
     (cd /usr/src/kernels/%{KVRA}%{?1:.%{1}} &&\
      /usr/bin/find . -type f | while read f; do\
-       hardlink -c /usr/src/kernels/*.fc*.*/$f $f\
+       hardlink -c /usr/src/kernels/*.%{?dist}.*/$f $f\
      done)\
 fi\
 %{nil}
-- 
1.7.1